lib/project/pro_motion/activities/pm_navigation_activity.rb

```diff
@@ -24,6 +24,13 @@ def on_options_item_selected(item)
     end
 
     def open_fragment(frag, options={})
+      if(options && options.length > 0)
```

Comment:
Shouldn't need this, since options should always be a hash. You can remove this conditional entirely.